app/move/views/view.njk

```diff
@@ -38,6 +38,17 @@
         }) if cancellationReason
       }
     }) }}
+  {% else %}
```

Comment:
I wonder if we just use the same banner style? The grey banner in the ticket looks like an old version of the design that @Andy-Millar and I ended up switching to the blue variation to convey status.

I know we talked about using it for "completed" but the app didn't have that concept at the time.